package.json

```diff
@@ -12,15 +12,16 @@
     "install-broker": "cd packages/openactive-broker-microservice && npm install",
     "install-interface-criteria": "cd packages/test-interface-criteria && npm install",
     "install-tests": "cd packages/openactive-integration-tests && npm install",
-    "install": "npm run install-broker && npm run install-interface-criteria && npm run install-tests",
+    "install": "npm run install-interface-criteria && npm run gen-interface-criteria-types && npm run install-broker && npm run install-tests",
```

Comment:
Why is `gen-interface-criteria-types` part of install if `npm run gen-types` is in the pre-commit hook? Should we be able to rely on the pre-commit hook here?

Reply:
Ah my mistake. From when I was testing the TIC changes in broker and I did not yet know that NPM file dependencies were symbolic links rather than copied files. I can remove that separately

Reply:
Just to note that this is also causing the build of the `openactive-cert-validator` to fail

```
       > openactive-test-suite@1.0.0 gen-interface-criteria-types /tmp/build_d08b62b2_
       > cd packages/test-interface-criteria && npm run gen-types
       
       
       > @openactive/test-interface-criteria@1.0.0 gen-types /tmp/build_d08b62b2_/packages/test-interface-criteria
       > copyfiles --up 1 "src/**/*.d.ts" built-types/ && tsc --project tsconfig.gen.json
       
       sh: 1: copyfiles: not found
```